packages/expo-barcode-scanner/src/BarCodeScanner.tsx

```diff
@@ -133,15 +132,16 @@ export class BarCodeScanner extends React.Component<BarCodeScannerProps> {
   };
 
   convertNativeProps(props: BarCodeScannerProps) {
-    const newProps = mapValues(props, this.convertProp);
-    return newProps;
-  }
+    return Object.entries(props).reduce<BarCodeScannerProps>((prev, [key, value]) => {
```

Comment:
don't really want to spread reduce() throughout the codebase. it is often not used for good reasons. simpler and also faster to `const nativeProps = {}; for (const key in props) { nativeProps[key] = transform(props[key]); } return nativeProps;` (less memory allocation).